dashboards/dataflow/README.md

```diff
@@ -0,0 +1,11 @@
+### Dashboards for Dataflow
+
+#### Notes
+
+- The full list of Dataflow Job metrics can be found [here](https://cloud.google.com/monitoring/api/metrics_gcp#gcp-dataflow).
+- Individual metric data can be viewed in [Metrics Explorer](find-dataflow-job-metrics.png).
```

Comment:
This content is carried into the console, but the image and relative link will not be - can you remove this line altogether?
```suggestion

```

Reply:
Ah ok, I didn't realize that. Thanks for the clarification!